samples/net/sockets/http_server/prj.conf

```diff
@@ -78,3 +78,10 @@ CONFIG_NET_TCP_TIME_WAIT_DELAY=0
 # Device drivers
 CONFIG_GPIO=y
 CONFIG_LED=y
+
+# Network debug config
+CONFIG_LOG_MODE_IMMEDIATE=y
```

Comment:
I would actually not enable this, it can slow down the device. Also it might make sense to remove it from http-client `prj.conf` file.

Reply:
Fixed.